test_schemas/en/test_relationships_unrelated.json

```diff
@@ -512,41 +512,150 @@
+                                                    "mandatory": true,
+                                                    "type": "Radio",
+                                                    "options": [
+                                                        {
+                                                            "label": "Yes",
+                                                            "value": "Yes",
+                                                            "action": {
+                                                                "type": "RemoveUnrelatedRelationships"
+                                                            }
+                                                        },
+                                                        {
+                                                            "label": {
+                                                                "placeholders": [
+                                                                    {
+                                                                        "placeholder": "person_name",
+                                                                        "transforms": [
+                                                                            {
+                                                                                "transform": "concatenate_list",
+                                                                                "arguments": {
+                                                                                    "list_to_concatenate": {
+                                                                                        "source": "answers",
+                                                                                        "identifier": ["first-name", "last-name"],
+                                                                                        "list_item_selector": {
+                                                                                            "source": "location",
+                                                                                            "id": "list_item_id"
+                                                                                        }
+                                                                                    },
+                                                                                    "delimiter": " "
+                                                                                }
+                                                                            }
+                                                                        ]
+                                                                    }
+                                                                ],
+                                                                "text": "No, none of these people are related to {person_name}"
```

Comment:
It would good to have these obviously different as I didn't pick up the placeholder in the second variant initially and then thought they were the same, meaning I didn't think the schema was exercising the bug

Reply:
The text was lifted from the census schemas, do you have a suggestion for something different that makes sense?

Reply:
Like I said it's so minor it can stay like it is, and why I approved. I just prefer test_schemas to be obvious what it's trying to demonstrate

" No, {person_name} is not related to anyone else in the household"

Reply:
I'm going to stick with the original text as it is consistent with the other "No" answer in referring to the list of people that appears on this page.